MakeYourCake.Services.Api/appsettings.Development.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Add this file to git ignore and delete it.

Reply:
fixed